miso-web/src/main/webapp/scripts/hot_index.js

```diff
@@ -0,0 +1,66 @@
+HotTarget.index = {
+  getCreateUrl: function() {
+    return Urls.rest.indices.create;
+  },
+  getUpdateUrl: function(id) {
+    return Urls.rest.indices.update(id);
+  },
+  requestConfiguration: function(config, callback) {
+    callback(config)
+  },
+  fixUp: function(index, errorHandler) {
+    if (index.family.fakeSequence) {
+      index.sequence = index.name;
+    }
+  },
+  createColumns: function(config, create, data) {
+    return [{
+      header: 'Family',
+      data: 'familyName',
+      readOnly: true,
+      include: true,
+      unpack: function(index, flat, setCellMeta) {
+        flat.familyName = index.family ? index.family.name : config.indexFamily.name;
+      },
+      pack: function(index, flat, errorHandler) {
+        if (!index.family) {
+          index.family = config.indexFamily;
+        }
+      }
+    }, HotUtils.makeColumnForInt("Position", true, 'position', HotUtils.validator.integer(true, 1, 2)),
+        HotUtils.makeColumnForText('Name', true, 'name', {
+          validator: HotUtils.validator.requiredTextNoSpecialChars
+        }), HotUtils.makeColumnForText('Sequence', !config.indexFamily.fakeSequence, 'sequence', {
+          validator: HotUtils.validator.requiredTextNoSpecialChars
+        }), {
+          header: 'Sequences',
+          data: 'realSequences',
+          description: 'Values should be comma-separated. e.g. "ACACAC, GTGTGT"',
+          include: config.indexFamily.fakeSequence,
+          validator: HotUtils.validator.requiredTextNoSpecialChars,
```

Comment:
Can this prohibit commas if it's not a fake sequence family?

Reply:
I think it's fair to expect administrators to enter values that make sense, but we could... If we're prohibiting commas, then we may as well prohibit a lot of other things though. Is there any reason to allow characters other than `[ACGT]`?

Reply:
@apmasell those are "fake sequence families" though, so they wouldn't be validated here

Reply:
Ah yes, 10X indices need to have both real sequences and fake sequences. The fake sequence is the name that the *Ranger software uses to demultiplex (SI-GA-A1 -> SI-GA-H12, not zero-padded) and the real sequences are what we use to avoid index collisions. Both will need to be able to be entered.

Reply:
Ah, I didn't know the fake sequence was actually used for anything. I'll make that column always appear and stop copying the name into the sequence. Any reason to prohibit commas specifically then?

Reply:
Relatedly, right now for "multi-sequence indices" the value in the HoT "Name" column is being stored in the `Indices.sequence` db column which doesn't look quite right.

Reply:
I think it would work for non-ACGT characters to be prohibited in the `Sequence`/`sequence` column above, and commas should be kept for this `Sequences`/`realSequences` column. I'm trying to think of good names to distinguish between the two columns though, if they'll both be displayed for the `fakeSequences` indices.

Reply:
* Changed the column name to "Fake Sequence" if a fake sequence family
* For non-fake-sequence families, validator is changed to only allow ACGT characters
* For fake-sequence families, "Sequences" field now only allows comma-separated sets of ACGT characters
* Added more info to field descriptions